code/modules/events/fake_virus.dm

```diff
@@ -5,10 +5,10 @@
 
 /datum/round_event/fake_virus/start()
 	var/list/fake_virus_victims = list()
-	for(var/mob/living/carbon/human/H in shuffle(GLOB.player_list))
-		if(!H.client || H.stat == DEAD || H.InCritical())
+	for(var/mob/living/carbon/human/victim in shuffle(GLOB.player_list))
+		if(!victim.client || victim.stat == DEAD || HAS_TRAIT(victim, TRAIT_CRITICAL_CONDITION))
```

Comment:
I migth be wrong but is client not always valid if its in the player list or is there edge cases where the client is already gone but the mob is still in player list?

Reply:
There's nothing more volatile than `client`, which can be nulled during a proc execution, but you are right. I'll remove it because on sane code this shouldn't happen.

Reply:
yeah i wasn't sure myself I havn't worked much with client code.